main.go

```diff
@@ -40,18 +40,23 @@ func create() *cli.CLI {
 		Args:           args,
 		HiddenCommands: []string{},
 		Commands: map[string]cli.CommandFactory{
-			"version":          command.CreateVersionFactory(version, commit),
-			"login":            command.LoginFactory,
-			"dataset":          cmd.DatasetFactory(ui),
-			"dataset upload":   cmd.DatasetUploadFactory(ui),
-			"dataset download": cmd.DatasetDownloadFactory(ui),
-			"dataset list":     cmd.DatasetListFactory(ui),
-			"dataset delete":   cmd.DatasetDeleteFactory(ui),
-			"job":              cmd.JobFactory(ui),
-			"job run":          cmd.JobRunFactory(ui),
-			"job list":         cmd.JobListFactory(ui),
-			"job logs":         cmd.JobLogsFactory(ui),
-			"job delete":       cmd.JobDeleteFactory(ui),
+			"version":             command.CreateVersionFactory(version, commit),
+			"login":               command.LoginFactory,
+			"dataset":             cmd.DatasetFactory(ui),
+			"dataset upload":      cmd.DatasetUploadFactory(ui),
+			"dataset download":    cmd.DatasetDownloadFactory(ui),
+			"dataset list":        cmd.DatasetListFactory(ui),
+			"dataset delete":      cmd.DatasetDeleteFactory(ui),
+			"job":                 cmd.JobFactory(ui),
+			"job run":             cmd.JobRunFactory(ui),
+			"job list":            cmd.JobListFactory(ui),
+			"job logs":            cmd.JobLogsFactory(ui),
+			"job delete":          cmd.JobDeleteFactory(ui),
+			"cluster":             cmd.ClusterFactory(ui),
+			"cluster list":        cmd.ClusterListFactory(ui),
+			"cluster list-config": cmd.ClusterListConfigFactory(ui),
```

Comment:
Having both a list and a list-config command is confusing to me, didn't see this in the design document right?

Reply:
I added this one because of the difference between clusters and contexts. I don't know what is the best option. I think it's valuable to be able to list the clusters separately from the contexts / configs, but I agree it's a bit confusing